src/ralph/settings/base.py

```diff
@@ -297,3 +297,11 @@ def os_env_true(var, default=''):
     INSTALLED_APPS += (
         'ralph.dns',
     )
+
+
+ENABLE_HERMES_INTEGRATION = os.environ.get('ENABLE_HERMES_INTEGRATION', False)
```

Comment:
please add here `HERMES = {'ENABLED': False}` too - without this pyhermes will try to publish events